ruby/red-arrow/test/test-slicer.rb

```diff
@@ -484,4 +484,38 @@ def setup
 7	   256	true   
     TABLE
   end
+
+  sub_test_case "MatchSubstringOptions family" do
+    def setup
+      @string_table = Arrow::Table.new(
```

Comment:
I think that `string_` is redundant. Because we process only string in this context.

Reply:
Sure, thanks.